packages/react-components/src/index.css

```diff
@@ -1,16 +1,16 @@
-:root {
-    --o-ui-disabled-opacity: 0.4;
-    --o-ui-pill-border-radius: 999px;
-    --o-ui-rounded-border-radius: var(--scale-alpha);
-    --o-ui-circular-border-radius: 100%;
-}
-
 [class^="o-ui-"] {
     --o-ui-focus-ring-inset-sm: 0 0 0 1px var(--o-ui-focus-ring-color);
     --o-ui-focus-ring-inset-md: 0 0 0 2px var(--o-ui-focus-ring-color);
     --o-ui-focus-ring-inset-lg: 0 0 0 4px var(--o-ui-focus-ring-color);
```

Comment:
We do have an `outset` and an `inset` focus-ring. Therefore I would expect to have var names that contains `inset` & `ouset`. We do have vars for `inset` but I don't see any for `outset`.

Reply:
That is because the outset focus ring is not using a box shadow

Reply:
Well as far as I understand all this we do have a var like `--o-ui-focus-ring-offset: 2px;` that exist to be used as part of an `outset` focus-ring?

`--o-ui-focus-ring-outset-offset: 2px`

Reply:
After reviewing other comments it should be something like:

```
--o-ui-focus-ring-outset-outer-offset: 2px;
--o-ui-focus-ring-outset-inner-offset: 1px;
```